README.md

```diff
@@ -47,6 +48,8 @@ new globalizePlugin({
 
 *output* is the name scheme of the built files.
 
+*transformExclusions* an array of functions to test on filepaths, and optionally exclude matching files from the dependency transformations. See [react-globalize-webpack-plugin](https://github.com/rxaviers/react-globalize-webpack-plugin) for an example usage.
+
```

Comment:
Is react-globalize-webpack-plugin itself the example, or is there an example supposed to be there?